internal/captain/command.go

```diff
@@ -347,7 +355,10 @@ func (c *Command) interceptFunc() InterceptFunc {
 // and add a warning banner for those who have.
 func (c *Command) SetUnstable(unstable bool) *Command {
 	c.unstable = unstable
-	c.cobra.Hidden = unstable
+	if !c.cfg.GetBool(constants.UnstableConfig) {
+		c.cobra.Hidden = unstable
+	}
+	c.name = fmt.Sprintf("%s (Unstable)", c.cobra.Use)
```

Comment:
This feels error prone. The `Command.name` isn't exclusively meant for the help output. I'd rather we address this in the help template.

Reply:
That's fair. We have a helper function that populates the `state` command's children. Moved this logic there.